ProjectSource/app-library/src/test/java/com/kayako/sdk/android/k5/messenger/replyboxview/ReplyBoxFragmentTest.java

```diff
@@ -0,0 +1,350 @@
+package com.kayako.sdk.android.k5.messenger.replyboxview;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v7.app.AppCompatActivity;
+import android.text.Editable;
+import android.view.KeyEvent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.TextView;
+import com.kayako.sdk.android.k5.R;
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertNotNull;
```

Comment:
✅ Good to see static resources imported instead of using FQN (fully-qualified-name) form in the code. 

This usage reduces verbosity and increases readability.

**No change suggested, looks good!**	  

<a id="8559bc70-53ca-11e8-b777-83ccc384eaae" href="none" codereview-uuid-reserved-tag />